chem/module_optical_averaging.F

```diff
@@ -3716,15 +3719,17 @@ subroutine optical_prep_gocart(nbin_o, chem, alt,relhum,          &
 !      WRITE(*,*)'         ','       ',(dlo_sectm(n),n=1,nbin_o)
 !      WRITE(*,*)'         ','       ',(dhi_sectm(n),n=1,nbin_o)
         dustfrc_goc8bin=0.
-        dlogoc=0.46*2.E-6 ! Begin lower dust bin, makes upper limit diam 20 micron diameter 
+        !dlogoc=0.46*2.E-6 ! Begin lower dust bin, makes upper limit diam 20 micron diameter 
```

Comment:
@saneku 
Alexander,
There are three of these lines that you commented out. It would probably be cleaner to remove the lines completely.

Reply:
done